packages/kotti-button-group/src/ButtonGroup.vue

```diff
@@ -20,3 +20,27 @@ export default {
 	},
 }
 </script>
+<style lang="scss">
+.button-group {
+	/** Fix for inline element space
+	https://css-tricks.com/fighting-the-space-between-inline-block-elements/ **/
+	font-size: 0;
+
+	.kt-btn {
```

Comment:
Should be `.kt-button` so that no second-guessing “was this class shortened or not” needs to happen.